glean-core/csharp/Glean/Metrics/LabeledMetricType.cs

```diff
@@ -102,9 +105,14 @@ public sealed class LabeledMetricType<T> where T : ILabeledSubmetricInterface
                 // required in order to make the compiler not complain. Since the `where` clause for
                 // this class cannot list more than one class, we need all our supported subtypes to
                 // implement a common interface and use that interface as the T type constraint. This
-                // allows us to then explicitly cast back to T, which is otherwise impossible.
+                // allows us to then explicitly cast bach to T, which is otherwise impossible.
```

Comment:
Oops, copypasta ;)
```suggestion
                // allows us to then explicitly cast back to T, which is otherwise impossible.
```

Reply:
Are you listening to Tocatta and Fugue?  Bach keeps coming bach, er, back ;)

Reply:
Ahahahahah :-D You got me!